libibumad/umad.h

```diff
@@ -81,7 +81,10 @@ typedef struct ib_mad_addr {
 	uint8_t gid_index;
 	uint8_t hop_limit;
 	uint8_t traffic_class;
-	uint8_t gid[16]; /* network-byte order */
+	union {
+		uint8_t	 gid[16]; /* network-byte order */
+		ib_gid_t ib_gid;
```

Comment:
Ah...  yea I missed that this is in the public header...  I wonder if we can deprecate that access somehow?